src/PublicApiAnalyzers/UnitTests/DeclarePublicAPIAnalyzerTestsBase.cs

```diff
@@ -2641,11 +2825,11 @@ public class D { }
         [Fact]
         public async Task MissingType_Unordered_CAsync()
         {
-            var source = @"
-public class B { }
-public class {|RS0016:{|RS0016:C|}|} { }
-public class D { }
-";
+            var source = $$"""
+                {{EnabledModifierCSharp}} class B { }
```

Comment:
I haven't looked at the test changes in this file. I am assuming these are just replacements and not any semantic changes.